### ISSUE TITLE
[beA] Priorisierung der Zuordnung von Nachrichten des beA ändern

### ISSUE THREAD

**Issue by RobertT01** [jlawyerorg/j-lawyer-org#1256]
Bitte die Priorisierung der Zuordnungskriterien von Nachrichten beim beA ändern.

Es wäre meines Erachtens sinnvoll die bestehende Priorisierung

  beA und Mail teilen sich die selbe Logik:

 - schauen ob der Absender in der Adressverwaltung bekannt ist
 - wenn ja: Ermitteln aller Akten, in denen der Kontakt als
 Beteiligte(r) geführt ist
 - Bilden von zwei Kategorien: nicht archiviert / archiviert
 - nicht archiviert nach oben, archiviert nach unten
 - innerhalb der Kategorien: nach AZ sortieren

so zu ändern, dass beim beA

innerhalb der Kategorien: nach AZ sortieren

zuerst geschaut wird, ob ein Empfängeraktenzeichen angegeben ist, wenn ja, ob es im System vorhanden ist und wenn ja, dass diese Akten als erste angezeigt wird. 

So sollte die wahrscheinliche genauste Zuordnung möglich sein und man würde die entsprechende Funktion im beA sinnvoll nutzen.

**Comment by stm9x9** [comment]
spricht etwas dagegen, das sogar an erster Stelle zu setzen? Also egal, ob der Absender irgendwie zu einer Akte paßt? ggf bekommt man von der - nach Klageeinreichung noch unbekannten - gegnerischen Kanzlei etwas per beA direkt zugestellt. Da hat man dann den Absender noch nicht in der Akte, aber die nennen dann mein Akz.

Wenn dann keine passende Akte gefunden werden konnte, dann bietet es sich möglicherweise an, nach Beteiligten zu suchen und dann zusätzlich in deren "Zeichen"-Feld in den ermittelten Akten nach dem Absender-Aktenzeichen aus der beA-Nachricht zu suchen.

![image](https://user-images.githubusercontent.com/29924198/120228697-f2fdff00-c24b-11eb-826b-584a4c6b4339.png)

kommt immer mal wieder vor, daß gerade Gerichte das Aktenzeichen des Anwalts nicht oder nicht korrekt eingeben

oder auch Behörden schreiben gerne mal nur ihre Akz rein.

**Comment by stm9x9** [comment]
Und als Ergänzung - oder soll ich lieber ein neues Ticket dafür machen? - wenn es gelänge, den Betreff einer Mail nach dem in jl hinterlegten Muster für Aktenzeichen zu durchsuchen und dann auch bei Mails priorisiert nach dem Aktenzeichen die Aktenvorschläge zu machen ...

**Comment by j-dimension** [comment]
> den Betreff einer Mail nach dem in jl hinterlegten Muster für Aktenzeichen zu durchsuchen und dann auch bei Mails priorisiert nach dem Aktenzeichen die Aktenvorschläge zu machen ...

Definitiv separates Ticket, ggf. ist das auch gar nicht zuverlässig umsetzbar, da weitreichende Freiheitsgrade bei der AZ-Syntax bestehen.

**Comment by Dickmaulruessler** [comment]
Eine Frage dazu: Wie erfolgt unter beA die Suche der dazugehörigen Akte, wenn man zusätzlich zu den jl-Aktenvorschlägen eine freie Sucheingabe macht? Das scheint keine UND Kombination i.S. einer Eingrenzung der Vorschläge zu sein. Korrekt? 
Wäre das sinnvoll? Falls ja: Gehört das hierher oder Extra-Ticket dafür?

**Comment by j-dimension** [comment]
> Wie erfolgt unter beA die Suche der dazugehörigen Akte, wenn man zusätzlich zu den jl-Aktenvorschlägen eine freie Sucheingabe macht?

Das verstehe ich nicht. Du kannst entweder den einen oder den anderen Knopf suchen. Oder verstehst Du die Suche als eine Suche innerhalb der Vorschläge? Das würde m.E. keinen Sinn ergeben. 

Ganz generell verhält sich die Suche exakt so wie die Suche nach Akten unter "Akte" - "suchen". Es ist die selbe Funktionalität.

**Comment by Dickmaulruessler** [comment]
> 
> Oder verstehst Du die Suche als eine Suche innerhalb der Vorschläge? Das würde m.E. keinen Sinn ergeben.
> 

Ja, so war mein Gedanke... Möglicherweise liege ich damit falsch... 

Gedanke war, dass man die Vielzahl der (an und für sich schon guten) Vorschläge schnell und leicht eingrenzen kann, z. B. mit Anfang des Manantennamens

**Comment by j-dimension** [comment]
Dann hätte man keine Wahl aus dem "Kreis der Vorschläge" auszubrechen und außerhalb der Vorschläge zuzuordnen.

**Comment by Dickmaulruessler** [comment]
Und / oder Button?

Für mich ist das jetzt sehr gut handhabbar - da einfach noch nicht so viel per beA kommt und man sich dafür Zeit nehmen kann.

Später werden wir von den Gerichten aber damit zugeballert werden und dann wären mehrere Suchoptionen - je nach Vorlieben des Anwenders - sicherlich praktisch.

**Comment by stm9x9** [comment]
> Später werden wir von den Gerichten aber damit zugeballert werden und dann wären mehrere Suchoptionen - je nach Vorlieben des Anwenders - sicherlich praktisch.

Da es in der Regel genau eine wirklich passende Akte gibt, wäre eine zielgenaue Auswahl durch jl praktischer als Suchoptionen :-)

**Comment by Dickmaulruessler** [comment]
Der Anteil an beA-Nachrichten ist mit der Zeit immer höher geworden. 
Um die Zuordnung zu erleichtern, wird dieses feature mit der Zeit daher immer wichtiger. 
Das (wohl) jetzt wichtigste Kriterium für die Zuordnungsvorschläge (Absender) ist jedenfalls im Bereich des beA nur bedingt hilfreich, wenn man Nachrichten ganz überwiegend von wenigen Amtsgerichten bekommt. 

Ich weiß nicht, was Jens hier vorschlagen kann, was machbar ist, 
aber sollte dieses Thema - vor allem für beA - im Nutzerkreis vielleicht noch einmal breiter diskutiert werden?
